docs/using.adoc

```diff
@@ -198,51 +197,34 @@ vagrant ssh
 You are now at a shell prompt inside the Vagrant box. You can now
 execute commands and use the tools provided.
 
-You can use the
-https://github.com/projectatomic/adb-utils/blob/master/README.adoc[sccli]
-to manage the orchestration services inside of ADB. `sccli` makes it
-easy to start and stop orchestration providers like Kubernetes or
-OpenShift.
+=== Using orchestrators
 
+You can use
+https://github.com/projectatomic/adb-utils/blob/master/README.adoc[sccli]
+to manage the orchestration services inside ADB. With `sccli`, you can start, stop,
+restart, and get the status of orchestration providers like OpenShift or Kubernetes.
 [[using-docker]]
 === Using `docker`
```

Comment:
I would suggest to get rid of the two sections i.e. docker and orchestrators and mention `sccli` for docker, OpenShift and Kubernetes.